execution/inc/leatherman/execution/execution.hpp

```diff
@@ -430,6 +434,7 @@ namespace leatherman { namespace execution {
      */
     std::tuple<std::string, std::string> process_streams(
             bool trim,
+            bool strip_cr,
```

Comment:
@MikaelSmith Do you have any thoughts or additional context on this? This feels like a "safe" change to me given how specific to the guts of execution this function actually is.